### ISSUE TITLE
Embedding BRLTTY in Cosmos OS for braille output (and maybe speech)

### ISSUE THREAD

**Issue by JordyDew** [CosmosOS/Cosmos#1578]
As a blind computer user, I'm bound to using synthetic speech and Braille output. That is a thing that seems to be missing in Cosmos OS.

Now, I have to use the OCR functionality of my screen reader over the VMware Workstation Player, but that means I have to re-run that process every time I want to check whether there has changed something in the output or not. That is a time consuming process and I cannot see what I type immediately.

I'm curious whether this could be added? I was thinking of BRLTTY, as that is being used in Linux OS'es and it is fully open-source.

**Comment by zarlo** [comment]
it could be added but will not likely as its a native program and native programs are not safe to run in cosmos (might be one day) and we cond have a libc it can link too but a c#/x# port could be made but there are some things cosmos it it current state still lacks for it to work
TL;DR it would not likely be done any time soon but might some day

**Comment by MishaTY** [comment]
Speech cannot yet be added because Cosmos does not have a audio driver yet and speech requires lots of work.

**Comment by quajak** [comment]
Yeah, while it would be awesome to support speech output and allow blind computer users to use, adding such features is very far in the future as we require a more solid base.
